docs/contributing/commit-signing.mdx

```diff
@@ -16,61 +16,144 @@ For example, on 3 August 2022, Stephen Lacy
 massive malware attack on GitHub by noticing unverified commits (i.e. commits that were not
 digitally signed).
 
-To protect against commit spoofing, all Bitwarden contributors are encouraged to digitally sign
-their commits.
+To protect against commit spoofing, <strong>all Bitwarden contributors are encouraged to digitally
+sign their commits</strong>.
 
 ## Setting up commit signing
 
-Github supports commit signing with GPG, SSH and S/MIME.
+GitHub supports [commit signing][github-verification] with GPG, SSH and S/MIME.
 
 :::tip
 
-If you're unsure what to use, we recommend you create a commit signing key using SSH. Remember to
-protect the key with a [strong passphrase or password][password-generation]. Alternatively set up
-your [yubikey as your signing key](#hardware-backed-ssh-keys).
-
-```bash
-# path to save your private signing key
-KEY_FILE=~/.ssh/bw-signing
-ssh-keygen -f $KEY_FILE -C "$(git config --global --get user.email)" -t ed25519
-```
+If you're unsure what to use, <strong>we recommend you create a commit signing key using SSH per
+latest security best practices </strong> (see the
+[PGP problem](https://www.latacora.com/blog/2019/07/16/the-pgp-problem/) for more details).
 
 :::
 
-### Configuration
+### Standard SSH Key Configuration
```

Comment:
I think we're trying to standardize on lowercase heading titles, after the first word - so "Standard SSH key configuration".  Can you fix the other titles on here as well?

Reply:
8d1bc61acc7295367f84f79457ce206bb35b3639